<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Feature Properties</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/plugins/RECS/recsicon.PNG</normaloff>:/plugins/RECS/recsicon.PNG</iconset>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>441</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Block Number</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Object ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Perimeter comp.(m)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Area comp.(sq m)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Street Number</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Perimeter measured(m)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Area measured(sq m)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Street Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Perimeter documented(m)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Woreda</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Area documented(sq m)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Surveyed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Modification</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Valid from date</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Valid to date</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Voided</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Verification</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_9"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEdit_11"/>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="lineEdit_12"/>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="lineEdit_15">
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QLineEdit" name="lineEdit_18">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLineEdit" name="lineEdit_19">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Parcel Type(*)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Unique Parcel ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>undefined</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>urban</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>administrative</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>railway</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>recreation and sports centre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>road/street</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>commercial</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>educational</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>others</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>forest</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mixed use</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>museum and cultural centre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>rural</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>publicSpace</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>religious institution</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>residence</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>industry</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>river</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>transport and parking</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>590</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
